verifier/src/error.rs

```diff
@@ -49,7 +50,7 @@ impl From<der::Error> for Error {
 
 impl From<serde_json::Error> for Error {
     fn from(e: serde_json::Error) -> Self {
-        Error::Serde(e)
+        Error::Serde(e.to_string())
```

Comment:
I contemplated adding a test like
```rust
#[cfg(test)]
mod test {
    use super::*;

    #[test]
    fn serde_error_to_string() {
        let bad_json = "not json";
        let e = serde_json::from_str::<serde_json::Value>(bad_json).unwrap_err();
        let err = Error::from(e);
        assert_eq!(err.to_string(), "Error parsing TCB(Trusted Computing Base) json info: expected ident at line 1 column 2");
    }
}
```

but it feels too fragile to changes in serde's error string formatting, and that's really what a test for this should be testing that the serde error string is propagated properly.
I'm not opposed to adding it if it's desired.

Reply:
What about something like
```Rust
err.to_string().contains(e.to_string())
```

Reply:
Good suggestion.
Added a test case similar to that suggestion in, https://github.com/mobilecoinfoundation/attestation/pull/170/commits/fcde25983083e9c37d21a73bb423f123eb7b6bd9